tests/unit/helpers/dates/companiDates.test.js

```diff
@@ -49,14 +59,6 @@ describe('DISPLAY', () => {
 
       expect(result).toBe('Le mercredi 24 novembre 2021 à 08h12 et 8 secondes');
     });
-
```

Comment:
t'as pas remplacé ce test comme t'as fait pour les autres ( "should return error if invalid argument" ligne 93, ligne 142 ...)

Mais en fait, est-ce que c'est encore pertinent de re-tester chaque fonction avec un cas d'argument non valide ?
Pourquoi ne pas faire confiance a luxon genre ?